apps/test-app/frontend/src/components/tree-widget/TreeWidget.tsx

```diff
@@ -172,21 +177,32 @@ export function StatelessTreeWidget(props: Omit<Props, "rulesetId">) {
       const parent: HierarchyNode | undefined = node ? (node as any).__internal : undefined;
       try {
         if (modelsTreeHierarchyProvider) {
-          return (await modelsTreeHierarchyProvider.getNodes({ parentNode: parent })).map(parseTreeNodeItem);
+          return (await modelsTreeHierarchyProvider.getNodes({ parentNode: parent, hierarchyLevelSizeLimit })).map(parseTreeNodeItem);
         }
         return [];
       } catch (e) {
         // eslint-disable-next-line no-console
         console.error(e);
```

Comment:
the error should not be logged if it gets handled

Reply:
Moved.